src/share/poudriere/common.sh

```diff
@@ -8328,6 +8337,10 @@ POUDRIERE_DATA="$(get_data_dir)"
 if [ -e "${POUDRIERE_DATA}" ]; then
 	POUDRIERE_DATA=$(realpath "${POUDRIERE_DATA}")
 fi
+JAIL_BASEMNT="$(get_jail_base_mnt)"
```

Comment:
`get_jail_base_mnt` is not needed. The following will provide a default and use whatever a user overrides with.
```
: ${JAIL_BASEMNT:=${POUDRIERE_DATA}/.m}
```

Reply:
Fixed.